nautobot/extras/tests/test_api.py

```diff
@@ -2023,44 +2000,33 @@ class JobResultTest(
     @classmethod
     def setUpTestData(cls):
         jobs = Job.objects.all()[:2]
-        job_ct = ContentType.objects.get_for_model(Job)
-        git_ct = ContentType.objects.get_for_model(GitRepository)
 
         JobResult.objects.create(
             job_model=jobs[0],
             name=jobs[0].class_path,
-            obj_type=job_ct,
             date_done=now(),
             user=None,
             status=JobResultStatusChoices.STATUS_SUCCESS,
-            data={"output": "\nRan for 3 seconds"},
-            task_kwargs=None,
+            task_kwargs={},
             scheduled_job=None,
-            task_id=uuid.uuid4(),
         )
         JobResult.objects.create(
             job_model=None,
```

Comment:
Is this right still? Or should it be linking to a system job for git repo sync now?

Reply:
Now that git repo sync is done by a regular job I think that we're covered by the other 2 job results. I do think there should be a job result for a job model that's been deleted. Unless there's something being tested here that I'm overlooking I'll change the job result name to `Deleted Job`.